static/app/components/links/listLink.tsx

```diff
@@ -1,5 +1,5 @@
 import * as React from 'react';
-import {Link, withRouter, WithRouterProps} from 'react-router';
+import {Link as RouterLink, withRouter, WithRouterProps} from 'react-router';
```

Comment:
would it be possible to use our link here too?

Reply:
It's somewhat difficult to add `onlyActiveOnIndex` to our link, so I think it's fine for now 🤷